### PR TITLE
Fix the sort by queue name

### PR DESCRIPTION
**Pull request by jipiboily**
Closes https://github.com/QueueClassic/queue_classic_admin/issues/29

Once reviewed, we need to:

- merge
- create tag
- cut a new gem (0.2.1)

/cc @smathieu @abevoelker

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -106,27 +106,208 @@ GEM
...
       activesupport (>= 3.0)
-      sprockets (~> 2.8)
+      sprockets (>= 2.8, < 4.0)
     sqlite3 (1.3.7)
     thor (0.19.1)
     thread_safe (0.3.4)
     tilt (1.4.1)
     treetop (1.4.15)
       polyglot
+      polyglot
```
Comment (smathieu) [inline Gemfile.lock]: ?

Comment (jipiboily) [inline Gemfile.lock]: huh, weird! I just ran bundle install...looking.

Comment (jipiboily) [inline Gemfile.lock]: Manually removed it. Re-ran bundle and didn’t do it again. Pushing now...

**Comment by jipiboily** [comment]
@smathieu I fixed the weird `Gemfile.lock` issue.

**Comment by smathieu** [comment]
:ship:

File: Gemfile.lock (hunk 1 above)
Comment (abevoelker) [inline Gemfile.lock]: FWIW I had that same weird thing happen. I actually had dozens of lines of "polyglot."  Did the same "fix" - manually reverted the change, after which it didn't happen again after rerunning `bundle`.  Might be a bundler bug.

**Comment by jipiboily** [comment]
Merged and I shipped 0.2.1 to Rubygems.

**Pull request merged**